File: Assets/Smash Way/Scripts JS/Menu/OpenWorld.js
#pragma strict

var Music:GameObject;
var Fon:RectTransform;
private var Stop:boolean=false;

function Start () {

}

function Update(){
if(!Stop){
if(Music.GetComponent.<AudioSource>().volume<1){
Music.GetComponent.<AudioSource>().volume=Music.GetComponent.<AudioSource>().volume+0.02;
Fon.GetComponent.<UI.Image>().color.a=Fon.GetComponent.<UI.Image>().color.a-0.02;
}else{Stop=true;gameObject.SetActive(false);}
}
}

File: Assets/Smash Way/Scripts CS/Menu/OpenWorld.cs
using UnityEngine;
using System.Collections;
using UnityEngine.UI;

public class OpenWorld : MonoBehaviour {

	public GameObject Music;
	public RectTransform Fon;
	private bool Stop=false;

	void Update(){
		if(!Stop){
			if(Music.GetComponent<AudioSource>().volume<1){
				Music.GetComponent<AudioSource>().volume=Music.GetComponent<AudioSource>().volume+0.02f;
				Fon.GetComponent<Image> ().color = new Color (Fon.GetComponent<Image>().color.r,Fon.GetComponent<Image>().color.g,Fon.GetComponent<Image>().color.b,Fon.GetComponent<Image>().color.a - 0.02f);
			}else{Stop=true;gameObject.SetActive(false);}
		}
	}
}
